setup.py

```diff
@@ -6,20 +6,31 @@
 with open('README.rst') as f:
     readme = f.read()
 
-with open('LICENSE') as f:
-    license = f.read()
+install_requires = ['hvac==0.2.17+git.b817da4',
+                    'requests',
+                    ]
+dependency_links = ['https://github.com/mracter/hvac/archive/b0811f0446760125baefea6383711035104e6df1.zip'
+                    '#egg=hvac-0.2.17+git.b817da4']
 
 setup(
     name='vaultkeeper',
     version='0.0.1',
-    description='A daemon that fetches and renews '
-                + 'Vault credentials for Gunicorn-on-Docker'
-                + 'Django applications',
+    description='An agent that fetches and renews '
+                'Vault credentials for Gunicorn-on-Docker'
```

Comment:
This is missing a space at the end of the line. Also, is this still necessarily "Gunicorn-on-Docker Django" stuff?